microk8s-resources/wrappers/microk8s-helm3.wrapper

```diff
@@ -1,15 +1,3 @@
 #!/bin/bash
 
-set -eu
-
-export PATH="$SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH"
-ARCH="$($SNAP/bin/uname -m)"
-export LD_LIBRARY_PATH="$SNAP/lib:$SNAP/usr/lib:$SNAP/lib/$ARCH-linux-gnu:$SNAP/usr/lib/$ARCH-linux-gnu"
-
-if [ ! -f "${SNAP_DATA}/bin/helm3" ]; then
-  echo "Helm 3 not available, try enabling it with 'microk8s enable helm3'" >&2
-  exit 0
-fi
-
-source $SNAP/actions/common/utils.sh
-"${SNAP_DATA}/bin/helm3" --kubeconfig="${SNAP_DATA}"/credentials/client.config "$@"
+"${SNAP}/bin/helm" --kubeconfig="${SNAP_DATA}"/credentials/client.config "$@"
```

Comment:
I think that by moving the helm binary from {SNAP_DATA} to {SNAP} we will make it report as disabled due to

Reply:
https://github.com/canonical/microk8s-core-addons/pull/70

Reply:
Community addons are also using the helm binary from  {SNAP_DATA}.

When users refresh to 1.25 those addons will probably fail because the addons repo do not refresh along. For backwards compatibility lets copy the helm binary on {SNAP_DATA} in the configure/post-refresh hook. WDYT?

Reply:
copy the binary, or can we get away with a symlink (pointing to `current`)?